src/components/ActionButton.tsx

```
@@ -0,0 +1,111 @@
+    setShowTooltip(false);
+  };
+
+  const getTooltipPositionStyles = () => {
+    switch (tooltipPosition) {
+      case 'bottom':
+        return { top: '100%', left: '50%', transform: 'translateX(-50%) translateY(8px)' };
+      case 'left':
+        return { top: '50%', right: '100%', transform: 'translateY(-50%) translateX(-8px)' };
+      case 'right':
+        return { top: '50%', left: '100%', transform: 'translateY(-50%) translateX(8px)' };
+      case 'top':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)